README.md

```diff
@@ -1105,48 +1134,33 @@
 
 ## Type Casting & Coercion
 
-  - Perform type coercion at the beginning of the statement.
-  - Strings:
-
-    ```javascript
-    //  => this.reviewScore = 9;
-
-    // bad
-    var totalScore = this.reviewScore + '';
-
-    // good
-    var totalScore = '' + this.reviewScore;
-
-    // bad
-    var totalScore = '' + this.reviewScore + ' total score';
-
-    // good
-    var totalScore = this.reviewScore + ' total score';
-    ```
-
-  - Use `parseInt` for Numbers and always with a radix for type casting.
-
-    ```javascript
-    var inputValue = '4';
-
-    // bad
-    var val = new Number(inputValue);
-
-    // bad
-    var val = +inputValue;
-
-    // bad
-    var val = inputValue >> 0;
-
-    // bad
-    var val = parseInt(inputValue);
+  - Avoid implicit type coercion entirely
```

Comment:
I've already witnessed things going down really quickly from this kind of expression. Usually when beginners hit this code, they are either impressed by it and start overusing it, or they just don't understand what is happening. This is usually a less-expressive way of performing an implicit type conversion, so in the spirit of avoiding operations that generate side effect, I suggest we disallow implicity type coersion entirely.